### PR TITLE
Docs/main readme

### LINKED ISSUE #95: Deployment documentation
Update and/or add documentation on how it's supposed to be setup and configured.

Figure out how to cross-reference documentation we have in the project.

### PR DESCRIPTION
**Pull request by blankdots**
Update main readme with necessary documentation.

Fixes #95 

related update: https://github.com/neicnordic/neic-sda/pull/7

### CODE REVIEW

**Comment by mihai-sysbio** [comment]
Here are some random ideas:
- include a section with [allcontributors.org](https://allcontributors.org/)
- mention licenses with [fossa.com](https://fossa.com/)
- include a link to [contributing guidelines](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/setting-guidelines-for-repository-contributors)

**Comment by blankdots** [comment]
@mihai-sysbio thx for the suggestions. I think the contributing guidelines can be added as part of this PR but for the others I think it would be more beneficial to integrate them in different PRs - seem like more time need to be invested to include them.

**Comment by blankdots** [comment]
I will reuse the same contribution guideline we had since the beginning of the Tryggve project: https://github.com/neicnordic/neic-sda/blob/master/CONTRIBUTING.md

**Comment by mihai-sysbio** [comment]
Ahh I had forgotten about [community guidelines](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-community-guidelines).

**Comment by mihai-sysbio** [comment]
Ahh I had forgotten about the [code of conduct](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/adding-a-code-of-conduct-to-your-project).

**Comment by blankdots** [comment]
happy with anyone else writing that but feel free to create issue on those, as mentioned not everything needs to be part of this PR :)

**Comment by blankdots** [comment]
and if I remember correctly that contributing guideline also contains some code of conduct info.

**Comment by blankdots** [comment]
fossa.com seems commercial this might be a good alternative: https://github.com/google/go-licenses

**Comment by mihai-sysbio** [comment]
> fossa.com seems commercial this might be a good alternative: https://github.com/google/go-licenses

On their website Fossa says "we offer special plans for non-profit, educational institution, and open source project budgets." I can confirm that Metabolic Atlas is on the `free` plan, and we have the [analysis of `master` in the Readme](https://github.com/MetabolicAtlas/MetabolicAtlas/#licenses). What I enjoy most from their service is the complete hands-off automation.

**Comment by blankdots** [comment]
ok. then we need a issue to integrated with it :slightly_smiling_face:

File: README.md [hunk 1]
```diff
@@ -1,11 +1,53 @@
-# sda-pipeline
+# sda-pipeline
```
Comment (jbygdell) [inline README.md:1]:
```suggestion
# sda-pipeline
```

File: README.md [hunk 2]
```diff
@@ -1,11 +1,53 @@
...
+
+`sda-pipeline` is part of [NeIC Sensitive Data Archive](https://neic-sda.readthedocs.io/en/latest/) and implements the components required for data submission.
+It can be used as part of a [Federated EGA](https://ega-archive.org/federated) or as a isolated Sensitive Data Archive. 
+`sda-pipeline` was built with support for both S3 and POSIX storage.
+
+## Deployment
+
+Recommended provisioning method for production is:
+
+* on a [Kubernetes cluster](https://github.com/neicnordic/sda-helm/), using `kubernetes` and `helm` charts;
```
Comment (jbygdell) [inline README.md:21]:
```suggestion
* on a , `kubernetes cluster` using the [helm chart](https://github.com/neicnordic/sda-helm/);
```

**Review by jbygdell — APPROVED** [review]

**Review by pontus — APPROVED** [review]

**Pull request merged**